core/src/components/action-sheet/test/a11y/index.html

```diff
@@ -63,6 +67,39 @@ <h1>Action Sheet - A11y</h1>
           },
         });
       }
+
+      function presentAriaLabelButton() {
+        openActionSheet({
+          header: 'Header',
+          subHeader: 'Subtitle',
+          buttons: [
+            {
+              text: 'Close',
+              htmlAttributes: {
+                'aria-label': 'close button',
+                'aria-labelledby': 'close-label',
+              },
+            },
+          ],
+        });
+      }
+
+      function presentAriaLabelCancelButton() {
+        openActionSheet({
+          header: 'Header',
+          subHeader: 'Subtitle',
+          buttons: [
+            {
+              text: 'Cancel',
+              role: 'cancel',
+              htmlAttributes: {
+                'aria-label': 'cancel button',
+                'aria-labelledby': 'cancel-label',
```

Comment:
I don't understand the `aria-labelledby` usage here. Is there an element with `#cancel-label`?

Reply:
No I was just adding a test to make sure it passes more than one property